modules/ossm-excluding-namespaces-from-cluster-wide-mesh-cli.adoc

```diff
@@ -41,12 +46,13 @@ spec:
         istio-discovery: enabled <1>
     - matchExpressions:
       - key: kubernetes.io/metadata.name <2>
-        operator: NotIn
+        operator: In
         values:
         - bookinfo
         - httpbin
+        - istio-system
 ----
 <1> Ensures that the mesh discovers namespaces that contain the label `istio-discovery: enabled`. The mesh does not discover namespaces that do not contain the label.
```

Comment:
> The mesh does not discover namespaces that do not contain the label.

This is an inaccuracy. If a namespace matches _any_ of the matches (`matchExpressions` or `matchLabels`), then that namespace will be included in the mesh.

Reply:
done